app/Http/Requests/ProgramRequest.php

```diff
@@ -21,10 +23,43 @@ public function authorize()
      *
      * @return array
      */
-    public function rules()
+    public static function rulesStore($request)
     {
-        return [
-            'name' => 'required|unique:programs,name',
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
         ];
+
+        return Validator::make($request->all(),
+            [
+                'name' => 'required|string|unique:programs',
+            ], $messages
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.MultipleArguments</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 37</br>Only one argument is allowed per line in a multi-line function call